src/hotspot/share/classfile/verifier.cpp

```diff
@@ -688,11 +688,11 @@ void ClassVerifier::translate_signature(Symbol* const method_sig,
 }
 
 void ClassVerifier::create_method_sig_entry(sig_as_verification_types* sig_verif_types,
-                                            int sig_index, TRAPS) {
+                                            int sig_index) {
   // Translate the signature into verification types.
   ConstantPool* cp = _klass->constants();
   Symbol* const method_sig = cp->symbol_at(sig_index);
-  translate_signature(method_sig, sig_verif_types, CHECK_VERIFY(this));
```

Comment:
CHECK_VERIFY is not directly related to exceptions. Are there really no verification errors possible from all of these calls where CHECK_VERIFY has been removed?

Reply:
The calls where CHECK_VERIFY's were removed are to functions that deal with signatures.  These methods do not check for errors because the signatures have already been verified by classFileParser.cpp in functions such as verify_legal_method_signature() and verify_legal_field_signature().

Reply:
Okay - thanks for confirming that.